### DISCUSSION TITLE
Automatic views?

### DISCUSSION THREAD

**Discussion by viperblck** [TeaByte/telegram-views#41] [Q&A]
Is possible to receive views automatically once a new post is pubblished ? or it works just manually ?

**Comment by TeaByte** [comment]
it is possible but the script don't support that 
but still possible to do

**Reply by viperblck** [reply]
how to do ?

**Comment by kaaajdesign** [comment]
hey.i start the terminal and then codes.its seems to count views but it does not add any view in telegram.may you help me
![photo_2024-02-12_20-35-03](https://github.com/TeaByte/telegram-views/assets/156255475/e7fd4488-1d5f-4054-8f64-0024581cdb8f)

**Reply by Sadra7x** [reply]
Dm for telegram @sadrab
